reflectivity_ui/interfaces/plotting.py

```diff
@@ -622,18 +622,20 @@ def _set_ymax(ymax, ynormed):
             ymin = 1.5
             ymax = 1e-7
             ynormed = self.main_window.data_manager.active_channel.r[P0:PN]
+            directbeam = self.main_window.data_manager.find_active_direct_beam_id()
             if len(ynormed[ynormed > 0]) >= 2:
-                ymin = min(ymin, ynormed[ynormed > 0].min())
-                ymax = _set_ymax(ymax, ynormed)
-                self.main_window.ui.refl.errorbar(
-                    self.main_window.data_manager.active_channel.q[P0:PN],
-                    ynormed,
-                    yerr=self.main_window.data_manager.active_channel.dr[P0:PN],
-                    label="Active",
-                    lw=2,
-                    capsize=1,
-                    color="black",
-                )
+                if directbeam is None:
```

Comment:
would it help User to write a message such as `No reflectivity for the selected direct-beam data` in the plot area when `directbeam is None` evaluates to False? Some code like the lines 640 to 648